### DISCUSSION TITLE
Making a theme

### DISCUSSION THREAD

**Discussion by MEGA-Dawg68** [SethBurkart123/EvenBetterSEQTA#47] [General]
Hi, I didn't know where to put this so I put this here...
How do I test out my own themes I was making? do I need to compile and put it somewhere or something else?

**Comment by SethBurkart123** [comment]
So have you already made it? For actually adding the theme and testing it you'd have to go to the file `interface/src/assets/themes.tsx` which currently looks like this:
```tsx
import hacker from './themeCovers/hacker.jpeg'

const themes = [
  {
    name: "Hacker",
    url: "https://raw.githubusercontent.com/SethBurkart123/BetterSEQTA-Themes/main/themes/hacker.json",
    coverImage: <img className="object-cover object-center w-full h-full" src={hacker} />,
  },
];

export default themes;
```
All you have to do, is add another theme to that, the cover image is a react component, if you do the same thing with a cover image then it will show up the same way. If you are confused in what format to do the actual theme .json url (like my hacker.json) just take a quick look at it and hopefully it will make a bit of sense. I'm thinking of making a theme creator extension so that you and others can make them easily, but that's a bit later down the line.

**Comment by MEGA-Dawg68** [comment]
Hey, sorry I never got around to doing this. How would it work now that the themes.tsx has changed?

**Comment by MEGA-Dawg68** [comment]
I take that back. I managed to Frankenstein BetterSEQTA+ into running my theme, and well the results are amazing lol
 
![Screenshot from 2024-03-11 18-11-12](https://github.com/BetterSEQTA/BetterSEQTA-Plus/assets/140035326/dac3e292-e461-4ff3-b3c6-7df607d35769)

**Reply by SethBurkart123** [reply]
That is wonderful... 😆. I'll work on making it easier to create themes in the near future.
